boards/arm/nucleo_f446ze/Kconfig.board

```diff
@@ -0,0 +1,8 @@
+# STM32F446RE Nucleo board configuration
+
+# Copyright (c) 2018 Philémon Jaermann
```

Comment:
I'm not sure if it's right to change the date of someone else's copyright notice? I have added my own notice below his to indicate that the file was added (by me) in 2021, but I left philemon's notice in place because it was present in the file I used as a template (nucleo_f446re.dts).
And indeed, `git blame` shows later modifications to that original file that did not change philemon's copyright notice. So maybe the best thing would be to assign copyright to the Zephyr project rather than individuals?

Reply:
ok

Reply:
Please use your own copyright here. For this kinf of files which are typically copy/paste from each other and we retain the copyright of the one making the effort to validate and upstream.